openupgrade_scripts/scripts/product/18.0.1.2/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,109 @@
+product      / product.combo            / combo_item_ids (one2many)     : NEW relation: product.combo.item
+product      / product.combo            / company_id (many2one)         : NEW relation: res.company
+product      / product.combo            / name (char)                   : NEW required
+product      / product.combo            / sequence (integer)            : NEW hasdefault: default
+product      / product.combo.item       / combo_id (many2one)           : NEW relation: product.combo, required
+product      / product.combo.item       / company_id (many2one)         : NEW relation: res.company, isrelated: related, stored
+product      / product.combo.item       / extra_price (float)           : NEW hasdefault: default
+product      / product.combo.item       / product_id (many2one)         : NEW relation: product.product, required
+product      / product.document         / _order                        : _order is now 'sequence, name' ('name')
+
+# NOTHING TO DO
+
+product      / product.document         / sequence (integer)            : NEW hasdefault: default
+
+# DONE: create sequence matching name
+
+product      / product.pricelist        / _order                        : _order is now 'sequence, id, name' ('sequence asc, id asc')
+
+# NOTHING TO DO
+
+product      / product.pricelist        / discount_policy (selection)   : DEL required, selection_keys: ['with_discount', 'without_discount']
+product      / product.pricelist.item   / display_applied_on (selection): NEW required, selection_keys: ['1_product', '2_product_category'], hasdefault: default
+
+# DONE: added in pre-migration
+
+product      / product.pricelist.item   / price_markup (float)          : NEW isfunction: function, stored
+product      / product.product          / _order                        : _order is now 'is_favorite desc, default_code, name, id' ('priority desc, default_code, name, id')
+product      / product.product          / combo_ids (many2many)         : previously in module point_of_sale
+product      / product.product          / service_tracking (selection)  : previously in module sale_project
+
+# NOTHING TO DO
+
+product      / product.product          / standard_price (float)        : needs conversion to v18-style company dependent
+
+# DONE
+
+product      / product.product          / type (selection)              : now required
+product      / product.tag              / _order                        : _order is now 'sequence, id' ('id')
+product      / product.tag              / sequence (integer)            : NEW hasdefault: default
+product      / product.template         / _order                        : _order is now 'is_favorite desc, name' ('priority desc, name')
+product      / product.template         / combo_ids (many2many)         : previously in module point_of_sale
+product      / product.template         / detailed_type (selection)     : DEL required, selection_keys: ['consu', 'service']
```

Comment:
Here it should be mentioned that this will require some adaptation on subsequent modules, like `event`.